tests/Vimeo/VimeoTest.php

```diff
@@ -214,4 +214,40 @@ public function testUploadTexttrackWithNonExistedFile()
         // Act
         $result = $vimeo->uploadTexttrack('https://vimeo.com/241711006', './the_file_is_invalid', 'fake_track_type', 'zh_TW');
     }
+
+    /**
+     * @expectedException Vimeo\Exceptions\VimeoRequestException
+     */
+    public function testReplaceWithVideoUriAndMachineIdShouldReturnVimeoRequestException()
```

Comment:
Please remove these other three machine ID tests, or change them to invalid param tests like you did with `testUploadWithInvalidParamShouldReturnVimeoRequestException`.